### PR TITLE
Add e2e crc test for installer

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml [hunk 1]
```diff
@@ -927,6 +927,51 @@ presubmits:
...
+      - args:
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-password-file=/etc/report/password.txt
+        - --report-username=ci
+        - --target=e2e-crc
+        command:
+        - ci-operator
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
```
Comment (patrickdillon) [inline ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml:955]: should be `optional: true`

Comment (praveenkumar) [reply ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml:955]: @patrickdillon yes, I will update that.

**Pull request merged**